catalog/spatial/registry/pom.xml

```diff
@@ -92,6 +92,11 @@
                 <version>${project.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.codice.ddf.registry</groupId>
```

Comment:
Never mind. It was a context issue for me. I didn't see the collapsed content in the file and didn't realize this wasn't here as a dependency; rather, it was in the dependencyManagement section. It's fine.